plugins/plugin.proto

```diff
@@ -84,18 +96,29 @@ message PluginResponse {
   // status code.
   repeated string error = 1;
```

Comment:
I wonder if it will be simpler to just wrap these three fields in another message something like PerOpenAPIResponse. Some better name. This will allow errors to be tied to specific OpenAPI file.

Consider making the names plural since they are repeated fields (other parts of the file too).